src/tests/tile.test.ts

```diff
@@ -0,0 +1,34 @@
+import { Board } from "../logic/Board";
+
+const board = new Board();
+
+test("an inner tile returns false with isOuterTile", () => {
+  const tile = board.grid.getHex([0, 0, 0]);
+  tile && typeof tile.isOuterTile === "function"
```

Comment:
not a criticism but why do you always check the type of the functions?

Reply:
it seems to be the only workaround to not being able to add to the Hex class:

```
import { Hex } from "honeycomb-grid";

interface ITile extends Hex {
  fill?: string;
  isOuterTile?(): boolean;
  getFill?(): string;
  setFill?(state: string): void;
}

export default ITile;
```

If we don't use `?` here we get the following error message: `Type 'Hex' is missing the following properties from type 'ITile': fill, isOuterTile, getFill, setFill`